src/commands.ts

```diff
@@ -139,9 +139,10 @@ Cypress.Commands.add(
         if (res.error) {
           log.set(
             "message",
-            `${res.message}\n[See comparison](${LINK_PREFIX}${btoa(
-              JSON.stringify({ title, imgPath })
-            )})`
+            `${res.message}\n[See comparison](${LINK_PREFIX}${btoa(unescape(encodeURIComponent(JSON.stringify({
```

Comment:
I think we shouldn't use `escape`/`unescape` methods as they're [described as deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape).
Is `encodeURIComponent` not enough to fix this error?

Reply:
Hi @FRSgit, thanks also for your attention, I'll look into it further soon.

Reply:
Hi @FRSgit, after research, the best way is to implement an escape Base64, so I found this answer and followed it.
https://stackoverflow.com/a/26603875/15495687

Reply:
@iamyoki Thanks for doing the research - indeed it seems that this is the best approach. Personally I would think about rereleasing the `webtoolkit.base64` on npm ([the same way `webtoolkit.md5` is released](https://www.npmjs.com/package/webtoolkit.md5)) as it's nicer to be able to share this kind of functionalities with other libraries.
But this solution is well enough for now, so I'd say let's stick with it until somebody will be willing to refactor 😄 
Thanks again!